google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -104,6 +107,12 @@ def py(self):
     return Pipeline._imports + default_args + dag_definition + \
         task_definitions + up_steam_statements
 
+  @staticmethod
+  def _get_datetime_expr(date_string):
```

Comment:
Should this be called something like `_get_datetime_expr_str`?

Reply:
Renamed this now. We can't evaluate the expression because we want the string as-is so that it becomes part of the airflow python spec.